### ISSUE TITLE
Espanso Sync Tool window on wayland

### ISSUE THREAD

**Issue by Mte90** [federico-terzi/espanso#2156]
### Description

On Debian sid with Wayland I have the Espanso Sync tool window in all the monitor I have.
It could be handy to have in the task bar or atleast the window hidden as it is not possible to open it.

### Motivation

Because can be annoying this window.

### Have you tried building it? Or would you like it?

- [ ] I'd be willing to contribute this feature to Espanso myself.

**Comment by smeech** [comment]
Duplicate of :
#1776
#1722
#1619

Fixed in the current `dev` code, but you'd need to compile, until we can find a way to make the next release (certificate issues).

**Comment by Mte90** [comment]
Using the https://github.com/espanso/espanso/releases/tag/v2.2.2 there isn't anymore the window.
The problem is that the window now just flash on the screen:
![Schermata_20241128_114318](https://github.com/user-attachments/assets/b8b62bae-28e7-415f-a68a-d500757e8fbb)

I was able to take a screenshot when it was happening. It just happens every 3-4 seconds in the monitor where is the cursor.

**Comment by Mte90** [comment]
Also everytime appears I lost the focus.
I will switch back to the previous release in the meantime.

**Comment by AucaCoyan** [comment]
Thanks for the details! Reopening the issue

**Comment by Mte90** [comment]
So I tried espanso compiling with this patch https://github.com/espanso/espanso/pull/1982 I still have the window flash but atleast there is no focus.

**Comment by AucaCoyan** [comment]
I can't repro the bug. In your opinion, should #1982 be merged and we deal with the rainbow square later? (another issue, I mean)

**Comment by Mte90** [comment]
Yeah I think that in the meantime remove the focus is good but I am wondering if espanso detection system on what I am writing still works (right now in my case it isn't, also with various espanso versions, so I am not sure why, I suggest to do a test).

**Comment by asm0dey** [comment]
Any plans to release 2.2.2?

**Comment by AucaCoyan** [comment]
we are still dealing with #2126

**Comment by asm0dey** [comment]
Aah, this is what is called a "production release" :) Thanks :)
